src/main/java/com/amadeus/booking/FlightOrder.java

```diff
@@ -0,0 +1,64 @@
+package com.amadeus.booking;
+
+import com.amadeus.Amadeus;
+import com.amadeus.Params;
+import com.amadeus.Response;
+import com.amadeus.exceptions.ResponseException;
+import com.amadeus.resources.Resource;
+
+/**
+ * <p>
+ *   A namespaced client for the
+ *   <code>/v1/booking/flight-orders:flightOrderId</code> endpoints.
```

Comment:
Is this **class** for /v1/booking/flight-orders:flightOrderId or /v1/booking/flight-orders endpoints?

Reply:
For `/v1/booking/flight-orders/:flightOrderId`. Check [here](https://github.com/amadeus4dev/amadeus-java/blob/master/src/main/java/com/amadeus/shopping/HotelOffer.java) for a similar endpoint.